cpp/ps0001/cpp_ps0001_solution.cpp

```diff
@@ -1,10 +1,15 @@
-#include<iostream>
+#include <iostream>
 using namespace std;
 int main()
 {
-	int x=155, y=940, e=168, z;
-	z = x + y + e;
-	cout << "the solution is :\n" << z;
-	return 0;
+	// prompting user for 3 number to add 0/2
+	unsigned short num1, num2, num3, sum;
+	cin >> num1 >> num2 >> num3;

```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [ ] used `std::cin` in a right way to prompt user for 3 numbers to add